### PR TITLE
Solved #382

### PR DESCRIPTION
**Pull request by AdityaTeltia**
Issue of toggle in mobile is resolved
working

### CODE REVIEW

File: assets/js/main.js [hunk 1]
```diff
@@ -93,56 +93,78 @@ let header = $(`
...
+      <div class="contact-form">
+          <form>
+              <fieldset class="form-group text-center">
+                  <input type="text" class="form-control" id="inputName" placeholder="Enter Name" required>
+              </fieldset>
+              <fieldset class="form-group text-center">
+                  <input type="email" class="form-control" id="inputEmail" placeholder="Enter email" required>
+              </fieldset>
+              <fieldset class="form-group text-center">
+                  <textarea class="form-control" id="message" placeholder="Message" required></textarea>
```
Comment (smaranjitghose) [inline assets/js/main.js:157]: New change to be incorporated

File: assets/css/home.css [hunk 2]
```diff
@@ -212,211 +211,11 @@ a:hover {
   color: transparent;
 }
 
-/* Footer container */
-
-.webintern-footer {
-  display: flex;
```
Comment (smaranjitghose) [inline assets/css/home.css:218]: New change to be incorporated

**Review by smaranjitghose — CHANGES_REQUESTED** [review]
Nice work @AdityaTeltia

**Comment by smaranjitghose** [comment]
@AdityaTeltia Please pull the recent changes to the upstream and commit them. Exculding that your PR works fine!

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us improve.
+title: Bug report
```
[the remark this answers is not in the document]
Comment (AdityaTeltia) [reply .github/ISSUE_TEMPLATE/bug_report.md:4]: I have to remove this ? and all mentioned below 
@smaranjitghose

[the remark this answers is not in the document]
Comment (smaranjitghose) [reply .github/ISSUE_TEMPLATE/bug_report.md:4]: You don't need to do these things manually. Just do a git pull upstream master

[the remark this answers is not in the document]
Comment (AdityaTeltia) [reply .github/ISSUE_TEMPLATE/bug_report.md:4]: It is saying already up to date !

[the remark this answers is not in the document]
Comment (AdityaTeltia) [reply .github/ISSUE_TEMPLATE/bug_report.md:4]: I understood , will make the pr after sometime @smaranjitghose

[the remark this answers is not in the document]
Comment (AdityaTeltia) [reply .github/ISSUE_TEMPLATE/bug_report.md:4]: @smaranjitghose I have made a new PR from another ```temp3``` branch ! check that one I was having issue in this ! And remember to put the tag of SWOC !
Thankyou

**Pull request closed without merging**